### ISSUE TITLE
Now I get this error compile for Win

### ISSUE THREAD

**Issue by rgleason** [seandepagnier/rtlsdr_pi#22]
```
"C:\Users\Frederick\Documents\GitHub\rtlsdr_pi\build\rtlsdr_pi.vcxproj" (defaul
t target) (3) ->
(Link target) ->
  rtlsdrPrefs.obj : error LNK2019: unresolved external symbol "double __cdecl V
HFFrequencyMHZ(int,bool)" (?VHFFrequencyMHZ@@YANH_N@Z) referenced in function "
public: virtual void __thiscall rtlsdrPrefs::OnInfo(class wxCommandEvent &)" (?
OnInfo@rtlsdrPrefs@@UAEXAAVwxCommandEvent@@@Z) [C:\Users\Frederick\Documents\Gi
tHub\rtlsdr_pi\build\rtlsdr_pi.vcxproj]
  C:\Users\Frederick\Documents\GitHub\rtlsdr_pi\build\Release\rtlsdr_pi.dll : f
atal error LNK1120: 1 unresolved externals [C:\Users\Frederick\Documents\GitHub
\rtlsdr_pi\build\rtlsdr_pi.vcxproj]

    2 Warning(s)
    2 Error(s)
```

**Comment by rgleason** [comment]
**VHFFrequencyMHZ(int,bool)**

I guess Windows does not like bool?

**Comment by rgleason** [comment]
I Pulled Holjem's  commits as a PR to my repos:master and it compiles now. Thanks.

Build succeeded.

"C:\Users\Frederick\Documents\GitHub\rtlsdr_pi\build\ALL_BUILD.vcxproj" (default target) (1) ->
"C:\Users\Frederick\Documents\GitHub\rtlsdr_pi\build\rtlsdr_pi.vcxproj" (default target) (3) ->
(ClCompile target) ->
  C:\Users\Frederick\Documents\GitHub\rtlsdr_pi\src\rtlsdrPrefs.cpp(82): warning C4800: 'int' : forcing value to bool 'tru
e' or 'false' (performance warning) [C:\Users\Frederick\Documents\GitHub\rtlsdr_pi\build\rtlsdr_pi.vcxproj]
  C:\Users\Frederick\Documents\GitHub\rtlsdr_pi\src\rtlsdr_pi.cpp(546): warning C4800: 'int' : forcing value to bool 'true
' or 'false' (performance warning) [C:\Users\Frederick\Documents\GitHub\rtlsdr_pi\build\rtlsdr_pi.vcxproj]

    2 Warning(s)
    0 Error(s)

**Comment by rgleason** [comment]
The plugin loads into v4.8.8 and I can enable it. I got stuck on calibrate somehow when I opened prefs. Could not close calibration until I found the hidden in back progress bar. When closed I could close preferences ok and not crash.

Refresh my mind here please, what do it need to install to have this work for Windos?
Driver- ?
Program-? AIS something?

**Comment by rgleason** [comment]
Also I've downloaded  rtlsdr_winutilities.zip into "here"  I assume that is where I the git plugin repos is located  C:\Users\..\Documents\GitHub\rtlsdr_pi  and the extracted there, which results in a  

 /bin folder with aisecoder.exe and  several lib files and rtl_fm.exe.

When I open rtlsdr_pi in O v4.8.8  I can enable the plugin and go to preferences and I think the aisecoder.exe is supposed to appear in the dropdown menu but there is nothing there.

When I check for a programdata\opencpn\plugins\rtlsdr  folder with the \bin folder, it does not exist.
So then I tried putting  the path to the \bin folder into the plugin, but the dropdown is still blank.

Do I need to install the zdiag driver first?

![screenshot 96](https://user-images.githubusercontent.com/2363959/53699672-f11d0b00-3db8-11e9-828b-71780f4a3f9c.png)

![image](https://user-images.githubusercontent.com/2363959/53699678-f5e1bf00-3db8-11e9-9566-eb5b1b294131.png)
